admin/app/components/trainings/create-or-update-training-form.js

```diff
@@ -17,9 +17,19 @@ class Form {
   @tracked locale;
   @tracked editorLogoUrl;
   @tracked editorName;
+
+  constructor({ title, link, type, duration, locale, editorLogoUrl, editorName } = {}) {
+    this.title = title || null;
+    this.link = link || null;
+    this.type = type || null;
+    this.duration = duration || null;
```

Comment:
On perd la valeur par défaut qu'on avait avant.

<img width="388" alt="image" src="https://user-images.githubusercontent.com/5855339/220328657-705000cd-1fb5-4f76-bced-135de934cfee.png">

J'imagine qu'il faut 
```suggestion
    this.duration = duration || { days: 0, hours: 0, minutes: 0 };
```